bears/csv/CSVLintBear.py

```diff
@@ -9,6 +9,15 @@
 class CSVLintBear:  # pragma nt: no cover
     """
     Verifies using ``csvlint`` if ``.csv`` files are valid CSV or not.
+    The ``CSVLintBear`` has validation that checks:
+    - structural formatting of a CSV file
+    - delimiter-separated values (dsv) file accessible via URL, File,
+    or an IO-style object (e.g. StringIO)
+    - against CSV dialects
+    - against multiple schema standards; CSV on the Web
+
+    Find out more at:
+    <https://github.com/theodi/csvlint.rb/blob/master/README.md>
```

Comment:
Is this link necessary ?

Reply:
Not necessary but it can provide more information about the validation process. So should I remove it?

Reply:
I will recommend it to remove it, because the class description itself explain about the validation checks that are being performed. Also, the link you provided includes ruby installation. So, IMO it should be removed

Reply:
If it gives more information, then OK 👍